examples/one-rate-plan/main.tf

```diff
@@ -0,0 +1,83 @@
+##############################################################################
+# Resource Group
+##############################################################################
+
+module "resource_group" {
+  source = "git::https://github.com/terraform-ibm-modules/terraform-ibm-resource-group.git?ref=v1.0.5"
+  # if an existing resource group is not set (null) create a new one using prefix
+  resource_group_name          = var.resource_group == null ? "${var.prefix}-resource-group" : null
+  existing_resource_group_name = var.resource_group
+}
+
+##############################################################################
+# Get Cloud Account ID
+##############################################################################
+
+data "ibm_iam_account_settings" "iam_account_settings" {
+}
+
+##############################################################################
+# Create CBR Zone
+##############################################################################
+# Create COS instance and Key protect instance.
+# Create COS bucket-1 with:
+# - Encryption
+# - Monitoring
+# - Activity Tracking
+# - One Rate Plan & One Rate Active Bucket Storage Class
+module "cos_bucket" {
+  source                = "../../"
+  resource_group_id     = module.resource_group.resource_group_id
+  region                = var.region
+  cross_region_location = null
+  cos_instance_name     = "${var.prefix}-cos"
+  cos_tags              = var.resource_tags
+  bucket_name           = "${var.prefix}-bucket-1"
```

Comment:
Maybe `-bucket-one-rate` to avoid a collision with bucket-1 in other examples, although the prefix should take care of that.